pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -608,6 +685,38 @@ func (r *Reconciler) updateHubInstallerSet(ctx context.Context, installerSet *v1
 	return nil
 }
 
+func (r *Reconciler) getAndDeleteDbInstallerSet(ctx context.Context) error {
```

Comment:
`getAndDeleteDbInstallerSet` and `getAndDeleteInstallerSet` functions do more or less the same things. Can we have use only `getAndDeleteInstallerSet` in both places